components/centraldashboard/public/components/main-page.pug

```diff
@@ -0,0 +1,43 @@
+app-drawer-layout.flex(narrow='{{narrowMode}}')
```

Comment:
Enforce pug templates to 4 spaces, so tabbing is more emphasized (since it's a tab based syntax)

Reply:
Understood. I've updated this but I think this should eventually be something we lint for.

Reply:
I agree! Looks good